packages/block-editor/src/components/link-control/search-input.js

```diff
@@ -3,43 +3,31 @@
  */
 import { useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
-import { Button } from '@wordpress/components';
-import { LEFT, RIGHT, UP, DOWN, BACKSPACE, ENTER } from '@wordpress/keycodes';
+import { Button, Notice } from '@wordpress/components';
 import { keyboardReturn } from '@wordpress/icons';
-
 /**
  * Internal dependencies
  */
 import { URLInput } from '../';
 
-const handleLinkControlOnKeyDown = ( event ) => {
-	const { keyCode } = event;
-
-	if ( [ LEFT, DOWN, RIGHT, UP, BACKSPACE, ENTER ].indexOf( keyCode ) > -1 ) {
-		// Stop the key event from propagating up to ObserveTyping.startTypingInTextField.
-		event.stopPropagation();
-	}
-};
-
-const handleLinkControlOnKeyPress = ( event ) => {
-	const { keyCode } = event;
-
-	event.stopPropagation();
-
-	if ( keyCode === ENTER ) {
-	}
-};
-
 const LinkControlSearchInput = ( {
 	value,
 	onChange,
 	onSelect,
 	renderSuggestions,
 	fetchSuggestions,
 	showInitialSuggestions,
+	errorMessage,
 } ) => {
 	const [ selectedSuggestion, setSelectedSuggestion ] = useState();
 
+	/**
+	 * Handles the user moving between different suggestions. Does not handle
+	 * choosing an individual item.
+	 *
+	 * @param {string} selection the url of the selected suggestion
+	 * @param {Object} suggestion the suggestion object
```

Comment:
Are these parameter explanations supposed to be capitalized with a `.` at the end?

Reply:
Looks like `string` is always lowercase across codebase.

I've added full stops.